src/generateSpec.ts

```diff
@@ -88,14 +97,13 @@ export function getHeaderParams(route: IRoute): oa.ParameterObject[] {
         schema,
       }
     })
-    .value()
 
-  const headersMeta = _.find(route.params, { type: 'headers' })
+  const headersMeta = route.params.find((p) => p.type === 'headers')
   if (headersMeta) {
     const schema = getParamSchema(headersMeta) as oa.ReferenceObject
     headers.push({
       in: 'header',
-      name: _.last(_.split(schema.$ref, '/')) || '',
+      name: schema.$ref.split('/').slice(-1)[0] || '',
```

Comment:
```suggestion
      name: schema.$ref.split('/').pop() || '',
```
this works too for getting the last element

Reply:
`pop` used here 7993cc4